bookwyrm/views/admin/celery_status.py

```diff
@@ -50,3 +52,18 @@ def get(self, request):
             "errors": errors,
         }
         return TemplateResponse(request, "settings/celery.html", data)
+
+
+@require_GET
+# pylint: disable=unused-argument
+def celery_ping(request):
+    """Just tells you if Celery is on or not"""
+    try:
+        ping = celery.control.inspect().ping()
+        if ping:
+            return HttpResponse()
+    # pylint: disable=broad-except
+    except Exception:
+        pass
+
+    return HttpResponse(500)
```

Comment:
Oh yes, it should definitely be sending a status response of 500, not the string "500". Thank you! I didn't add anything too helpful to the message since it's a public endpoint